### DISCUSSION TITLE
Details not visible?

### DISCUSSION THREAD

**Discussion by CoMPaTech** [ESPresense/ESPresense-companion#410] [Q&A]
For some reason (already a few updates now) the 'details' do not show up when viewing a device in the companion (i.e. companion page -> devices -> device). The carrot is there to fold/unfold, but the list of details is gone. (And yes the circles at all are still visible). Any clues?

**Comment by DTTerastar** [comment]
Is it still happening?  Is there any errors in the web browser console?  Can you try a different browser?

**Comment by CoMPaTech** [comment]
Yeah, still happens on across devices, was wondering if it was just me (apparently so) so I'll do a dive into javascript console then when time allows

**Comment by CoMPaTech** [comment]
Multiple OS' and browsers used, not showing anywhere. No (apparent) clashes in the (browser) console, so I suspect it being something within the HA installation. I'll try to (also) install the companion on my HA development environment (which is mostly clean) to see/build-out to get to the issue

**Comment by CoMPaTech** [comment]
Tried on the dev instance, but nothing showing - so I'm thinking it's somewhere in MQTT then?

**Comment by CoMPaTech** [comment]
@DTTerastar not sure if this is relevant, but every now and then one or more devices keep 'hanging' as well. To better describe, I have 4 apple-based products that I've added as IRK devices, but even though 'last seen' is at 7 hours, today both my devices show with confidence 1 and 93 resp. and still both in a room/floor instead of `n/a`. 

Configuration is set to 'low'-ish (timeout 15 and away on 30 - still need to ramp these up).

**Comment by CoMPaTech** [comment]
Still happening, no clue what else to try :)

**Comment by CoMPaTech** [comment]
Just for reference, still doesn't work, no JS errors popping up, nothing. Is there something I could debug to see where the client details should be?

**Comment by CoMPaTech** [comment]
@DTTerastar that was pretty simple, just don't understand why; when I just pressed 'save' on the settings, it started working again ... does that make any sense? (I.e. click on devices, click a device, from map change to settings, change nothing, just save and presto
